<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenDRIVE System</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionMapWnd"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionPGDBSetting"/>
    <addaction name="actionLocalFilesDatabasePath"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionGPSDataLayer"/>
    <addaction name="actionTrajectoryLayer"/>
    <addaction name="actionOSMWayLayer"/>
    <addaction name="actionOpenDRIVERoadLayer"/>
   </widget>
   <widget class="QMenu" name="menuNavigate">
    <property name="title">
     <string>Navigate</string>
    </property>
    <addaction name="actionToLatLon"/>
    <addaction name="actionToXY"/>
   </widget>
   <addaction name="menuNavigate"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMapWnd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Map Window</string>
   </property>
  </action>
  <action name="actionLocalFilesDatabasePath">
   <property name="text">
    <string>Local Files Database Path</string>
   </property>
  </action>
  <action name="actionGPSDataLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GPS Data Layer</string>
   </property>
  </action>
  <action name="actionTrajectoryLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Trajectory Layer</string>
   </property>
  </action>
  <action name="actionOSMWayLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OSM Way Layer</string>
   </property>
  </action>
  <action name="actionOpenDRIVERoadLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenDRIVE Road Layer</string>
   </property>
  </action>
  <action name="actionToLatLon">
   <property name="text">
    <string>To Lat Lon</string>
   </property>
  </action>
  <action name="actionToXY">
   <property name="text">
    <string>To X Y</string>
   </property>
  </action>
  <action name="actionPGDBSetting">
   <property name="text">
    <string>PostgreSQL Database Setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
